grove/qft/tests/test_inverse_qft.py

```diff
@@ -0,0 +1,50 @@
+##############################################################################
+#
+#    A collection of tests for the inverse quantum Fourier transform.
+#
+#    Written by Aaron Vontell on January 28th, 2017
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+##############################################################################
+
+from pyquil.gates import X
+from grove.qft.fourier import *
```

Comment:
Its really nitpicky to list out all the gates, but you could do that too if you prefer